scipy/ndimage/_measurements.py

```diff
@@ -1535,10 +1541,11 @@ def center_of_mass(input, labels=None, index=None):
     >>> ndimage.center_of_mass(d)
     (inf,)
     """
-    normalizer = sum(input, labels, index)
+    input = np.asarray(input)
+    normalizer = sum_labels(input, labels, index)
     grids = np.ogrid[[slice(0, i) for i in input.shape]]
 
-    results = [sum(input * grids[dir].astype(float), labels, index) / normalizer
+    results = [sum_labels(input * grids[dir].astype(float), labels, index) / normalizer
```

Comment:
could you explain the change here?

Reply:
This is pure cosmetics: `ndimage.sum` is an backwards compat alias for `sum_labels`, so use a specific name and avoid shadowing the builtin.